cli/test/test_cmdskip.py

```diff
@@ -0,0 +1,103 @@
+from click.testing import CliRunner
+import unittest
+from popper_test import PopperTest
+import popper.commands.cmd_run as run
+import unittest
+import os
+import subprocess
+import sys
+
+FIXDIR = f'{os.path.dirname(os.path.realpath(__file__))}/fixtures'
+
+
+def _wfile(name, format):
+    return f'{FIXDIR}/{name}.{format}'
+
+
+class TestSkip(unittest.TestCase, PopperTest):
+
+    def test_skip(self):
+
+        workflow_file_loc = _wfile("skip", "workflow")
```

Comment:
Thanks for Reviewing the pr @ivotron, I will make it to YAML, I made it workflow for the time being because of the issues that some features are currently not working in YAML files so to prevent failure of the tests I have currently decided to keep it workflow.

Reply:
OK. We can leave `.workflow` for skip then.

Reply:
Yes, Sure.